dd-sdk-android/src/main/kotlin/com/datadog/android/event/SpanEventMapper.kt

```diff
@@ -7,10 +7,12 @@
 package com.datadog.android.event
 
 import com.datadog.android.tracing.model.SpanEvent
+import com.datadog.tools.annotation.NoOpImplementation
 
 /**
  * An interface which can be implemented to modify the writable attributes inside a SpanEvent.
  */
+@NoOpImplementation
 interface SpanEventMapper : EventMapper<SpanEvent> {
```

Comment:
Why do you  need to explicitely create this interface ? It doesn't add anything new to the existing EventMapper interface ?

Reply:
It is, as we decided it will not allow a `Nullable` return.